types.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import "time"
+
+// Would be nice to be able to do, instead of this:
+// httpinternal "github.com/lunarway/release-manager/internal/http"
```

Comment:
The `internal` package is special in Go and specifically prohibits imports from other modules.
We could however move them out into a shared package in `release-manager` to make it possible. :)

Reply:
Oh so that's why it wouldn't work